2.4/root/usr/share/container-scripts/mongodb/replset_petset_supervisor.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "${CONTAINER_SCRIPTS_PATH}/common.sh"
+
+info "Waiting for local MongoDB to accept connections ..."
+wait_for_mongo_up &>/dev/null
+
+# This is a full hostname that will be added to replica set
+# (for example, "replica-2.mongodb.myproject.svc.cluster.local")
+readonly MEMBER_HOST="$(hostname -f)"
+
+# Description of possible statuses: https://docs.mongodb.com/manual/reference/replica-states/
+readonly WAIT_PRIMARY_STATUS="while(rs.status().startupStatus || (rs.status().hasOwnProperty(\"myState\") && rs.status().myState != 1)) { printjson(rs.status()); sleep(1000); }; printjson(rs.status());"
+readonly WAIT_SECONDARY_STATUS="var mbrs; while(!mbrs || mbrs.length == 0 || mbrs[0].state != 2) { printjson(rs.status()); sleep(1000); mbrs = rs.status().members.filter(function(el){return el.name.indexOf(\"${MEMBER_HOST}:\") > -1}); }; print(mbrs[0].stateStr);"
+readonly ADD_MEMBER="if(rs.status().members.filter(function(el){return el.name.indexOf(\"${MEMBER_HOST}:\") > -1}).length==0){printjson(rs.add('${MEMBER_HOST}'));}else{print('{ \"ok\" : 1 }')}"
```

Comment:
+1 for multiple lines.

@php-coder @rhcarvalho In this PR you don't use "complete" functions for one action in mongod  (for example `mongo_add`, `mongo_remove`,... - they contain everything and are influenced by parameters). Instead of it you save peaces of JavaScript in variables and everytime you want to use it, you have to specify `mongo` + parameters + from variable assemble the code to evaluate.

The second approach is more frustrating when using (having to type `mongo` and parameters every time. On the other hand it is more complex - you only share code but can add whatever you want in each situation.

What it better and what we decide to use in mongodb scripts (these one and the old one)? (I think it should be consistent)  WDYT?